github-issue/check.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import requests
+import json
+import sys
+import os
+import datetime
+from pprint import pprint
+import dateutil.parser
+
+def _check(instream):
+    #payload = json.load(instream)
+    source = payload['source']
+    token = source['token']
+    repository = source['repository']
+now = datetime.datetime.now() - datetime.timedelta(seconds = 180)
+    date = now.isoformat()
+
+    try:
+        issueNumbers
+    except NameError:
+        issueNumbers = {}
+
+    headers = {'Authorization': 'token ' + token}
+    endpoint = "https://api.github.com/repos/" + repository + "/issues"
+    connection = requests.get(endpoint, headers=headers)
+    
+    if issueNumbers = {}:
+        for i in connection.json():
+            issueNumbers.update( {i['number'] : i['updated_at']})
```

Comment:
assuming this is to handle the case where no previous data is passed in, we want to be careful not to return a ton of stuff here, because it will immediately trigger jobs for every record, lets only return the most recent record in this case